Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -2084,9 +2095,26 @@ void vtkMRMLMarkupsNode::WriteMeasurementsToDescription()
       this->PropertiesLabelText += "\n" + measurementText;
       }
     }
+
+  if (description.empty())
+    {
+    this->PropertiesLabelText.clear();
```

Comment:
We may need to use the description for other things, so if no measurement is selected to be shown in description then just clear it once and then don't clear it every time (it would fix a problem in Path Explorer module in SlicerIGT).